python/uv.lock

```diff
@@ -0,0 +1,438 @@
+version = 1
```

Comment:
Please can we add support for `uv` in the separate PR ? 
We have an open issue for it: https://github.com/kubeflow/sdk/issues/20
cc @briangallagher @szaher